lib/classes/Admin/Editor.php

```diff
@@ -12,7 +12,7 @@ public function __construct()
 
     public function loadStyles()
     {
-        add_editor_style('assets/styles/output/editor.css');
+        add_editor_style('styles/output/editor.css');
```

Comment:
Editor styles seemed to stop working altogether. This seems to be a regression bug after #804.

Reply:
Oops. Fixed it now.